html/changelog.html

```diff
@@ -55,6 +55,42 @@
 		
 		-->
 		<div class="commit sansserif">
+			<h2 class="date">23 January 2016</h2>
+			<h3 class="author">Fox McCloud updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="rscdel">Removes Vampire HUD</li>
+				<li class="tweak">moves the Vampire blood counter to a more easy to see location</li>
+				<li class="rscadd">Adds in a Changeling chemical counter display</li>
+				<li class="rscadd">Adds in a Changeling sting counter display</li>
+				<li class="rscadd">Vampire total blood and usable blood will now appear under status</li>
+				<li class="tweak">Changes vampire blood display to be similar to ling chemical counter</li>
+				<li class="bugfix">Fixes unnecessary toxin damage being dealt on severe bloodloss</li>
+				<li class="bugfix">Fixes human mobs not receiving proper random names</li>
+				<li class="rscadd">Adds in knight armor</li>
+				<li class="rscadd">Grants the chaplain a set of crusader knight armor</li>
+				<li class="rscadd">Adds in cult-resistant ERT paranormal space suit</li>
+				<li class="rscadd">Allows the Chaplain to convert his null rod into a holy sword with crusader knight armor</li>
+				<li class="tweak">Chaplain's closet is now a secure closet</li>
+				<li class="rscadd">Adds in Assault Gear crate to cargo</li>
+				<li class="rscadd">Adds in military assault belt rcsadd: Adds in spaceworthy swat suits</li>
```

Comment:
Huh, that's odd. Something went wrong during the automatic changelog generation. Let me fix that.